adafruit_uc8151d.py

```diff
@@ -33,11 +34,18 @@
 _START_SEQUENCE = (
     # b"\x01\x05\x03\x00\x2b\x2b\x09"  # power setting
     # b"\x06\x03\x17\x17\x17"  # booster soft start
-    b"\x04\x80\xc8"  # power on and wait 10 ms
+    b"\x04\x80\xc8"  # power on and wait 200 ms
```

Comment:
@makermelissa  is it OK to wait 200ms for _both_ the b/w and the tri-color LCD?

Reply:
It's been this way for quite a while. I really only updated the commend to match the actual value. The additional delay is probably because CP is a little slower than Arduino. Either way it works.

Reply:
Also, the values are the same between the color and b/w displays in Arduino (10ms in that case).